### PR TITLE
OP-1217 Add Medical Stock Feature

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1217.
Paired with https://github.com/informatici/openhospital-core/pull/1338

### CODE REVIEW

File: src/main/java/org/isf/medstockmovtype/gui/MedicalDsrStockMovementTypeBrowserEdit.java [hunk 1]
```diff
@@ -288,9 +297,29 @@ private JComboBox<String> getTypeComboBox() {
 	}
 
 	/**
-	 * This method initializes jDataPanel	
-	 * 	
-	 * @return javax.swing.JPanel	
+	 * This method initializes typeTextField
+	 * 
+	 * @return javax.swing.JTextField
+	 */
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medstockmovtype/gui/MedicalDsrStockMovementTypeBrowserEdit.java:303]: These comments look wrong to me.

**Review by dbmalkovsky — COMMENTED** [review]
Just one note about a comment block.

File: src/main/java/org/isf/medstockmovtype/gui/MedicalDsrStockMovementTypeBrowserEdit.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalDsrStockMovementTypeBrowserEdit.java:303]: Fixed, thanks!

File: src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java [hunk 2]
```diff
@@ -106,8 +106,7 @@
 import com.github.lgooddatepicker.zinternaltools.WrapLayout;
 
 /**
- * MovStockBrowser - list medicals movement. let the user search for movements
- * 					  and insert a new movements
+ * MovStockBrowser - list medicals movement. let the user search for movements and insert a new movements
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java]:
```suggestion
 * MovStockBrowser - list medicals movement. Let the user search for movements and insert a new movement.
```

**Review by dbmalkovsky — APPROVED** [review]
LGTM

Just one small javadoc suggestion that can be ignored.

**Pull request merged**